src/pages/students/forms/paymentDataForm/PaymentData.tsx

```diff
@@ -1,77 +1,69 @@
 import { MainContainer, 
-        TitleContainer, 
         FormContainer, 
         Label, 
         Input, 
-        Title, 
         Button, 
         ButtonsContainer, 
         Select, 
         SmallInput, 
         InputsContainer,
-        Text} from "../styles/PaymentData.styles";
-import { PaymentPlanName } from "../../../app/types/models/PaymentPlanName";
+        Text} from "./PaymentData.styles";
+import { PaymentPlanName } from "../../../../app/types/models/PaymentPlanName";
 import { useForm } from "react-hook-form";
 import { yupResolver } from "@hookform/resolvers/yup";
 import * as yup from "yup";
-import { paymentDataScheme } from "../schemes/PaymentData.scheme";
-import { ErrorMessage } from "../../../components/header/ErrorMessage";
-import { useNavigate } from "react-router";
-import { setStudentData } from "../StudentRegistrationFormSlice";
-import type { RootState } from "../../../app/store/store";
-import { useDispatch, useSelector } from "react-redux";
+import { paymentDataScheme } from "./PaymentData.scheme";
+import { ErrorMessage} from "../../../../components/error_message/ErrorMessage"
+import type { PaymentDataProps } from "../../create-student/CreateStudent";
 
+interface FormProp {
+  createStudentCall: (data: PaymentDataProps) => void;
+  paymentData:PaymentDataProps | undefined;
+  stepBack:()=>void;
+}
 
-function PaymentData() {
+function PaymentData({
+  createStudentCall,
+  paymentData,
+  stepBack
+}: FormProp) {
 
-  type FormData = yup.InferType<typeof paymentDataScheme>;
 
-  const dispatch = useDispatch();
+  type FormData = yup.InferType<typeof paymentDataScheme>;
     
-  const studentRegistrationForm = useSelector((state: RootState) => state.studentRegistrationForm);
+  const studentRegistrationForm = paymentData ||
+  {
+    paymentPlanName:"",
+    classPrice:null,
+    extraClasses:null,
+    paymentDay:undefined,
+  };
```

Comment:
Podriamos hacer lo siguiente con esto


```suggestion
  const studentRegistrationForm = data || DEFAULT_PAYMENT_DATA
```

creando esto

```
const DEFAULT_PAYMENT_DATA: PaymentDataProps = {
  paymentPlanName: "",
  extraClasses: null,
  classPrice: null,
  paymentDay: undefined
};
```